webproject/src/main/java/com/google/sps/data/Room.java

```diff
@@ -35,6 +36,7 @@ private Room (Builder builder) {
         this.category = builder.category;
         this.minPrice = builder.minPrice;
         this.deliveryFee = builder.deliveryFee;
+        this.timestamp = builder.timestamp;
```

Comment:
does the PR need to have this file modified?

Reply:
How I implemented the notification is that the user will be notified when a room is created after the webpage has been loaded, to ensure that the user is not notified for rooms that they are already aware of or that existed before they accessed that webpage. That is why I added the timestamp for the creation of a Room and left it in the PR relating to notifications. Should I move it to a separate PR?

Reply:
got it. sorry I missed it.